window-placement.lisp

```diff
@@ -66,8 +93,15 @@
   (let ((match (rule-matching-window window)))
     (if match
         (destructuring-bind (group-name frame raise lock
-                             &key from-group create restore class instance type role title) match
-          (declare (ignore from-group lock class instance type role title))
+                             &key from-group create restore class class-not instance
+                               instance-not type type-not role role-not title title-not
+                               match-properties-and-function
+                               match-properties-or-function)
+            match
+          (declare (ignore from-group lock class class-not instance
+                           instance-not type type-not role role-not
+                           title title-not match-properties-and-function
+                           match-properties-or-function))
```

Comment:
Instead of adding variables to the ignore declaration, remove them from the d-bind.

Reply:
I just realized this recommendation was wrong. I forgot to add that we have to add `&allow-other-keys` otherwise destructuring-bind will raise an error. Sorry about that. Will commit the fix soon